ts/src/header-validator/source.test.ts

```diff
@@ -2108,7 +2177,7 @@ const testCases: TestCase[] = [
     name: 'trigger-data-too-many-within',
     json: JSON.stringify({
       destination: 'https://a.test',
-      trigger_data: Array.from({ length: 33 }, () => 0),
+      trigger_data: Array.from({ length: 33 }, (_, i) => i),
```

Comment:
Do we need to update spec for `trigger_data`: https://wicg.github.io/attribution-reporting-api/#parse-trigger-data-into-a-trigger-spec-map?

Reply:
`trigger_data` doesn't allow duplication, so should be fine.